### ISSUE TITLE
[Accessibility] Feed elements in navigation drawer have insufficient height

### ISSUE THREAD

**Issue by amkcpu** [FredJul/Flym#713]
One of the results of an accessibility scan with the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor) is that the feed elements in the navigation drawer have an insufficient height:

<img src="https://user-images.githubusercontent.com/46764284/97503500-6fa6c300-1975-11eb-902a-8312bfa1dd41.png" width=300 />

It is 45dp, while 48dp is recommended.

**Comment by amkcpu** [comment]
The same goes for the feed items in the "Reorder feeds" screen.

**Comment by andreacantone** [comment]
The search field in the "Search feed" screen also does not respected the touch target.
